helix-term/src/ui/picker.rs

```diff
@@ -166,7 +166,12 @@ impl<T: 'static> Component for FilePicker<T> {
         surface.clear_with(area, background);
 
         let picker_width = if render_preview {
```

Comment:
It would be less confusing to have the config as `preview-width` instead of `picker-width` since I would assume `picker-width` to mean the width of the entire window (including the preview and the picker).

Reply:
But `preview` is the right side not always showed. Maybe we have to choose a name for the left side of the picker. Suggestions?

Reply:
Then I think we can have two different options - `file-picker.width` and `file-picker.preview-width`. `width` can control the whole picker + preview window size and `preview-width` can control the size of the preview inside this window.

Reply:
Done :-)